support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -212,16 +215,24 @@ function AccountHolderNameInput(props: {
 	);
 }
 
-function RecaptchaInput(props: { id: string }) {
+function RecaptchaInput(props: {
+	setRecaptchaToken: (token: string) => void;
+	expireRecaptchaToken?: () => void;
+}) {
+	useRecaptchaV2(
+		recaptchaId,
+		props.setRecaptchaToken,
+		props.expireRecaptchaToken,
+	);
```

Comment:
We moved this hook inside of the recaptcha component so that we only call it when we also render the component it's going to mount to. The recaptcha won't display if we call this too early.